core/trino-main/src/main/java/io/trino/operator/join/JoinUtils.java

```diff
@@ -67,7 +67,7 @@ public static boolean isBuildSideReplicated(PlanNode node)
                             planNode -> planNode instanceof ProjectNode ||
                                     isLocalRepartitionExchange(planNode) ||
                                     isLocalGatherExchange(planNode))  // used in cross join case
-                    .where(joinNode -> isRemoteReplicatedExchange(joinNode) || isRemoteReplicatedSourceNode(joinNode))
+                    .where(planNode -> isRemoteReplicatedExchange(planNode) || isRemoteReplicatedSourceNode(planNode))
```

Comment:
Please update commit message to
`Rename ambiguous variable in JoinUtils`
We try to follow guidelines from https://cbea.ms/git-commit/ for writing commit messages

Reply:
updated